### ISSUE TITLE
importPreset throws errors on color 

### ISSUE THREAD

**Issue by chrsmlls333** [cocopon/tweakpane#504]
Using tweakpane 3.1.5

I'm seeing an error whenever I try to export & import a color variable. I thought it may be due to the formatting, but {rgba} objects and "rgba()" strings also report similar but different errors.

```
TypeError: value.getComponents is not a function 
    at colorToHexRgbaString

TypeError: value.toRgbaObject is not a function
    at writeRgbColorObject
```

```js
import { Pane } from 'tweakpane';

var state = { color: "#ffffff" }

var p = new Pane()
p.addInput(state, "color")

p.importPreset(p.exportPreset()) // TypeError: value.getComponents is not a function at colorToHexRgbaString
```

**Comment by cocopon** [comment]
My mistake... there is a problem reading the preset values. Fixed in the latest version 3.1.6.

**Comment by chrsmlls333** [comment]
Thank you for the quick fix!
